tox.ini

```diff
@@ -20,6 +20,7 @@ deps =
     transaction
     zc.lockfile
     zdaemon
+    zodbpickle
```

Comment:
Technically, you only need to list packages from `tests_require` here; things that are in `install_requires` will already be installed by tox.  (There might be one or two exceptions, e.g. if we `install_requires=[..., 'foo', ...]` and `foo` has a `setup_requires` on `bar`, and `bar` isn't already installed explicitly, we might get a `./.eggs/bar-*.egg` that is prevented if we explicitly list `bar` in deps here.)